pytition/petition/models.py

```diff
@@ -11,6 +11,7 @@
 from django.db import transaction
 from django.urls import reverse
 from django.utils import timezone
+from phonenumber_field.modelfields import PhoneNumberField
```

Comment:
Nitpicking here but I'm trying to keep the django imports in one block, and the other imports other blocks.
Could you put it with the block containing tinymce and colorfield?